File: iniciante/1010/1010.py
cod_1, n_pecas_1, valor_unitario_1 = input().split()
cod_1 = int(cod_1)
n_pecas_1 = int(n_pecas_1)
valor_unitario_1 = float(valor_unitario_1)

cod_2, n_pecas_2, valor_unitario_2 = input().split()
cod_2 = int(cod_2)
n_pecas_2 = int(n_pecas_2)
valor_unitario_2 = float(valor_unitario_2)


total = (n_pecas_1 * valor_unitario_1) + (n_pecas_2 * valor_unitario_2)

print(f"VALOR A PAGAR: R$ {total:.2f}")

File: iniciante/1010/1010.cpp
#include <iostream>
#include <iomanip>

int main()
{
  int cod_peca1, cod_peca2, n_pecas1, n_pecas2;
  double valor_peca1, valor_peca2, total;

  std::cin >> cod_peca1 >> n_pecas1 >> valor_peca1;
  std::cin >> cod_peca2 >> n_pecas2 >> valor_peca2;

  total = n_pecas1 * valor_peca1 + n_pecas2 * valor_peca2;

  std::cout << std::fixed << std::setprecision(2);
  std::cout << "VALOR A PAGAR: R$ " << total << std::endl;

  return 0;
}
